masterlist.yaml

```
@@ -22981,17 +22981,24 @@ plugins:
         nav: 4
 
   - name: 'WindhelmEntranceOverhaul.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19400/' ]
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19400/'
+        name: 'City Entrances Overhaul - Windhelm'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65092/'
+        name: 'Rob''s Bug Fixes - City Entrances Overhaul - Windhelm'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - City Entrances Overhaul - Windhelm](https://www.nexusmods.com/skyrimspecialedition/mods/65092/)' ]
+        condition: 'checksum("WindhelmEntranceOverhaul.esp", A91FF679)'
```

You probably also want to include the checksum for the Banners Fix plugin:

```suggestion
        condition: 'checksum("WindhelmEntranceOverhaul.esp", A91FF679) or checksum("WindhelmEntranceOverhaul.esp", 6972E572)'
```